webpack.js

```diff
@@ -7,7 +7,7 @@ const UglifyJSPlugin = require('uglifyjs-webpack-plugin')
 const ExtensionsPlugin = require('./extensions/extensions-plugin')
 
 const nodeConfig = {
-  devtool: process.env.NODE_ENV === 'production' ? 'source-map' : 'eval-cheap-module-source-map',
+  devtool: 'source-map',
```

Comment:
This slowed my build time from 10-15s to 30-35s in dev. What's your build time like?

Reply:
Cold prod compilation — 23s, dev — 33s.
My problem with the previous option was it gave me zero hints for the errors thrown from the generators (like transpiled async/await code).
Let's try to find the sweet spot, but before the change I basically wasn't able to trace the errors.

For the browser change I also wasn't able to step through the code in the debugger before, now it works like a charm.

Reply:
OK thanks, our build time are similar. I'll open a out-of-`x` issue about optimizing this. Let's keep it as is